edx_proctoring/templates/proctored_exam/footer.html

```diff
@@ -8,3 +8,72 @@
       </a>
   </p>
 </div>
+
+<script type="text/javascript">
+  var startProctoredExam = function(selector, exam_id, action_url, start_immediately, attempt_proctored) {
+    $.post(
+      action_url,
+      {
+        "exam_id": exam_id,
+        "attempt_proctored": attempt_proctored,
+        "start_clock": start_immediately
+      },
+      function(data) {
+        // reload the page, because we've unlocked it
+        location.reload();
+      }
+    ).fail(function(){
+      enableClickEvent(selector);
+      var msg = gettext(
+        "There has been a problem starting your exam.\n\n" +
+        "Possible reasons are that your account has not been fully activated,\n" +
+        "you have are experiencing a network connection problem, or there has been\n" +
+        "a service disruption. Please check these and try again."
```

Comment:
Replace the last sentence:
"Please check these and try again." -> "Check your account or network connection and try again."